client/actions/eventActions.js

```diff
@@ -99,7 +102,10 @@ export function modifyEvent(id, data) {
 }
 
 export function deleteCenterEvent(data) {
-  const { id, centerId } = data;
+  const {
+    id,
+    centerId
+  } = data;
   return (dispatch) => {
     dispatch({ type: 'DELETE_CENTER_EVENT' });
     axios.delete(`api/v1/events/${id}`).then((res) => {
```

Comment:
You might want to explicitly return your `axios.function()` calls whenever you first make a dispatch call right above them. It makes no difference now, but when you start writing tests for your actions using moxios, you would run into errors as a result of the `axios.function()` calls not being made after dispatch has been called.